day_6/exercises/person.rb

```diff
@@ -1,5 +1,69 @@
-# Create a person class with at least 2 attributes and 2 behaviors. 
+# Create a person class with at least 2 attributes and 2 behaviors.
 # Call all person methods below the class and print results
 # to the terminal that show the methods in action.
 
 # YOUR CODE HERE
+class Person
+  attr_reader :hair_color, :skin_color, :height, :weight
+  def initialize(hair_color, skin_color, height, weight)
+    @hair_color = hair_color
+    @skin_color = skin_color
+    @height = height
+    @weight = weight
+  end
+
+  def dye_hair=(c)
```

Comment:
always air on the side of verbosity and clarity. So I would name the parameter here `color` instead of `c`